src/ledger.js

```diff
@@ -0,0 +1,131 @@
+// LedgerSystem class
+const BigNumber = require('bignumber.js');
+const Data = require("./database");
```

Comment:
'Data' seems too generic and misleading for what this object is

Reply:
renamed to DataConnector